proto/v1/info.proto

```diff
@@ -60,4 +60,19 @@ message VariableGroupInfoRequest {
   // only contain the child variable (group) if it has data for at least one
   // of the entities.
   repeated string constrained_entities = 2;
+}
```

Comment:
For my understanding, is there no VariableGroupInfoResponse?

Reply:
It returns StatVarGroupNode directly

Reply:
That feels a bit inconsistent with other calls, like VariableInfoResponse, which return node + response?

Also, if we have a VariableGroupInfoResponse, perhaps BulkVariableGroupInfoResponse (like other responses) be a repeated of the former?

Reply:
That's a good point. I was hoping to keep original format but good to make it right at this round.